util/util.go

```diff
@@ -0,0 +1,36 @@
+package util
+
+import (
+	"bufio"
+	"os"
+	"path/filepath"
+)
+
+//GetFiles get all file path of a give root directory
+func GetFiles(root string) ([]string, error) {
+	var files []string
+	err := filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
+		if !info.IsDir() {
+			files = append(files, path)
+		}
+		return nil
+	})
+	return files, err
+}
+
+//GetWords Get all words in file
+func GetWords(filePath string) ([]string, error) {
```

Comment:
requirement is count specific words from input, so store all words in slice can cause memory issues.